core/corehttp/gateway_test.go

```diff
@@ -505,6 +505,9 @@ func TestIPNSHostnameBacklinks(t *testing.T) {
 	if !strings.Contains(s, k2.Cid().String()) {
 		t.Fatalf("expected hash in directory listing")
 	}
+	if !strings.Contains(s, "<button class=\"type-icon\" title=\"Copy CID\" onclick=\"copyText(") {
+		t.Fatalf("expected button to copy CID in directory listing")
+	}
```

Comment:
This test is not needed because all of what you are testing is just part of the template, that mean something else more important is already broken.

Reply:
test removed